public/app/features/admin/LicenseChrome.tsx

```diff
@@ -6,9 +6,7 @@ import { GrafanaTheme } from '@grafana/data';
 const title = { fontWeight: 500, fontSize: '26px', lineHeight: '123%' };
 
 const getStyles = stylesFactory((theme: GrafanaTheme) => {
-  const backgroundUrl = theme.isDark
-    ? '/public/img/licensing/header_dark.svg'
-    : '/public/img/licensing/header_light.svg';
+  const backgroundUrl = theme.isDark ? 'public/img/licensing/header_dark.svg' : 'public/img/licensing/header_light.svg';
```

Comment:
Won't this link to `$GRAFANA_ROOT/admin/public/...`? 🤔

Reply:
Ah, Torkel showed me that there's a `<base href="...">` in the `<head>` which makes relative links relative to that base instead of relative to where you are right now. Cool learning!